mak/COPY

```diff
@@ -93,6 +108,7 @@ COPY=\
 	$(IMPDIR)\core\sys\osx\mach\thread_act.d \
 	\
 	$(IMPDIR)\core\sys\osx\sys\cdefs.d \
+	$(IMPDIR)\core\sys\osx\sys\event.d \
```

Comment:
Would that be confusing though?  To me it seems better to consistently have imports directory for core.sys.osx match core.sys.darwin.  I think it was a mistake before that added event.d without copying it to import dir.